components/Snackbar/src/ColorThemer/MDCSnackbarColorThemer.m

```diff
@@ -18,10 +18,18 @@
 
 @implementation MDCSnackbarColorThemer
 
-+ (void)applyColorScheme:(nonnull id<MDCColorScheme>)colorScheme
-    toSnackbarMessageView:(nonnull MDCSnackbarMessageView *)snackbarMessageView {
-  // TODO(yar): Currently a stub. Will need to update the color setting once we have concluded
-  // the proper color theming and scheme.
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme {
+  MDCSnackbarManager.snackbarMessageViewBackgroundColor =
```

Comment:
@jverkoey let me know if you find a better way to approach opacity for our theming.